tsconfig.strictTypeCheck.json

```diff
@@ -0,0 +1,10 @@
+{
+  "extends": "./modules/tinymce/tsconfig.json",
+  "compilerOptions": {
+    "strict": true,
+    "noEmit": true
+  },
+  "include": [
+    "modules/tinymce/src/core/main/ts/ui"
```

Comment:
Added this here because Typescript complains if there is no input

Reply:
yes it doesn't surprised me we need to have at least one folder passing as strict to start using this config in CI.